readme.md

```diff
@@ -2522,3 +2522,17 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 Contributions are welcome and encouraged! Read the [contribution guidelines][contrib-guide] first.
 
 [contrib-guide]: contributing.md
+
+### [Diplomacy](https://boardgamegeek.com/boardgame/483/diplomacy) 
+
+> This classic game of pure negotiation has taken many forms over the years.
+
+The first The Avalon Hill Game Co version has perhaps the widest release, but Avalon Hill re-released the game in 1999, complete with a colorful new map and metal pieces. In 2008, Avalon Hill released a 50th anniversary edition with a new map and cardboard pieces representing the armies and navies.
```

Comment:
Please format the rest of the description block, they’ll need the leading `> ` for each line.

Reply:
I see the entry got moved, the only thing I see now is that both other paragraphs for the description needs to be formatted with the leading `>`.